game_of_thrones/app/views/characters/edit.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Edit the Character</h1>
+
+ <%= form_for @character do |f| %>
+   <%= f.label :name %>
+   <%= f.text_field :name %>
+
+   <%= f.label :age %>
+   <%= f.text_field :age %>
+
+   <%= f.label :img_url %>
+   <%= f.text_field :img_url %>
+
+   <p>House <%= select_tag :house_id, options_for_select(@houses) %></p>
+
```

Comment:
you can use `f.select :house_id, @houses` here as well-- that way you don't have to manually assign the house_id in the controller

this is leveraging how you've defined your `@houses` in your character controller edit method (the array of arrays of names and ids)